esp/public/media/scripts/onsite/ajax_status.js

```diff
@@ -772,13 +795,9 @@ function populate_classes()
     for (var i in data.catalog.categories)
     {
         var new_category = data.catalog.categories[i];
-        /*
-        //  Skip "open class" category
-        if (new_category.id == open_class_category.id)
-            continue;
-        */
+
         data.categories[new_category.id] = new_category;
-        if (settings.categories_to_display.indexOf(new_category.id) == -1)
+        if (settings.is_initial && (settings.categories_to_display.indexOf(new_category.id) == -1))
```

Comment:
I added this check to make the grid consistent with the settings when the data was refreshed (99e15d7 prevented the refresh from clobbering your settings, but it would still show all categories in the table regardless of what those settings were).